packages/main/src/ListItem.js

```diff
@@ -364,6 +369,14 @@ class ListItem extends ListItemBase {
 		return this.type === ListItemType.Detail;
 	}
 
+	get typeNavigation() {
+		return this.type === ListItemType.Navigation;
+	}
+
+	get navigated() {
```

Comment:
this getter is redundant, as "navigated" is part of the component metadata - there is already a getter with the same name